core/src/main/scala/kafka/admin/TopicCommand.scala

```diff
@@ -378,22 +379,22 @@ object TopicCommand extends Logging {
 
   def shortMessageSizeWarning(maxMessageBytes: Int): String = {
     "\n\n" +
-      "*****************************************************************************************************\n" +
-      "*** WARNING: you are creating a topic where the max.message.bytes is greater than the consumer ***\n" +
-      "*** default. This operation is potentially dangerous. Consumers will get failures if their        ***\n" +
-      "*** fetch.message.max.bytes < the value you are using.                                            ***\n" +
-      "*****************************************************************************************************\n" +
+      "******************************************************************************************************\n" +
+      "*** WARNING: you are creating a topic where the max.message.bytes is greater than the consumer     ***\n" +
+      "*** default. This operation is potentially dangerous. Consumers will get failures if their         ***\n" +
+      "*** max.partition.fetch.bytes < the value you are using.                                           ***\n" +
+      "******************************************************************************************************\n" +
       s"- value set here: $maxMessageBytes\n" +
-      s"- Default Consumer fetch.message.max.bytes: ${ConsumerConfig.FetchSize}\n" +
+      s"- Default Consumer max.partition.fetch.bytes: ${ConsumerConfig.MAX_PARTITION_FETCH_BYTES_CONFIG}\n" +
       s"- Default Broker max.message.bytes: ${kafka.server.Defaults.MessageMaxBytes}\n\n"
   }
 
   def longMessageSizeWarning(maxMessageBytes: Int): String = {
     "\n\n" +
       "****************************************************************************************************\n" +
-      "*** WARNING: you are creating a topic where the max.message.bytes is greater than the broker      ***\n" +
+      "*** WARNING: you are creating a topic where the max.message.bytes is greater than the broker     ***\n" +
       "*** default. This operation is dangerous. There are two potential side effects:                  ***\n" +
-      "*** - Consumers will get failures if their fetch.message.max.bytes < the value you are using     ***\n" +
+      "*** - Consumers will get failures if their max.partition.fetch.bytes < the value you are using   ***\n" +
```

Comment:
Can we use `ConsumerConfig.MAX_PARTITION_FETCH_BYTES_CONFIG` in the places we print `max.partition.fetch.bytes`. That way it helps track stale usages in the future.